src/app/core/components/navigation/theme.scss

```diff
@@ -18,5 +18,9 @@
   .navbar {
     background-color: map.get($colors, background-app-bar);
     box-shadow: map.get($colors, shadow);
+
+    .km-header-logo {
+      border-right: 1px solid map.get($colors, divider);
```

Comment:
One issue with this is that border-right is there even on login screen. I think its better to rename this class and apply it using `ngClass` when there is some value in `currentView` i.e. `currentView` value is displayed. See screenshot below:

![screenshot-localhost_8000-2022 06 08-17_07_03](https://user-images.githubusercontent.com/13975988/172612846-e80a96ca-6d6c-4b1a-bc43-acf2a9917b65.png)